source/troubleshooting.rst

```diff
@@ -0,0 +1,25 @@
+===============
+Troubleshooting/FAQ
```

Comment:
I think this should just be FAQ. It could also have a question of "what's the point" that briefly explains how Minecraft formatting works, perhaps with links to the wiki page on it?